src/App.vue

```diff
@@ -2,10 +2,12 @@
   <div id="app">
     <div class="noteList">
       <div class="addBtm"><b>+ Add note</b></div>
-      <div class="notes"></div>
+      <div class="notesListTi" v-for="(note, index) in notes" :key="index" @click="tap(index)">
+        {{ note.title }}
+      </div>
     </div>
-    <editor :key="index" :curr-data="currdate" />
-    <viewer :key="index" :preview-data="predata" />
+    <editor :key="index" :current-note="currentNote" />
+    <viewer :key="index" :preview-data="preNote" />
```

Comment:
It would be nice to make the name of the variable more meaningful.
```suggestion
    <viewer :key="index" :previous-note="previousNote" />
```